build_ee.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+USER=ameszaro-1
```

Comment:
You shouldn't let your username leak into the PR.

Reply:
ahh sorry, though I removed this